src/ts/ConvivaAnalytics.ts

```diff
@@ -631,6 +631,9 @@ export class ConvivaAnalytics {
 
```

Comment:
There is a comment in line 409 stating that the framerate is unused. Please also update this comment 🙂

Reply:
That line seems to be referring to encoded framerate. This PR is for reporting rendered framerate which calculates "Average FPS" for a session.
It seems like this metric does not fit into the same category being defined in that comment.

Reply:
ah oke 👍